modules/configuring-egress-proxy-edns-operator.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-configuring-cluster-wide-egress-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-cluster-wide-proxy_{context}"]
+= Configuring the External DNS Operator to trust the certificate authority of the cluster-wide proxy
+
+. Create the config map to contain the CA bundle in the `external-dns-operator` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator create configmap trusted-ca
+----
+
+. To inject the trusted CA bundle into the config map, add the `config.openshift.io/inject-trusted-cabundle=true` label to the config map by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator label cm trusted-ca config.openshift.io/inject-trusted-cabundle=true
+----
+
+. Update the subscription of the External DNS Operator by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator patch subscription external-dns-operator --type='json' -p='[{"op": "add", "path": "/spec/config", "value":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca"}]}}]'
+----
+
+. After the deployment of the External DNS Operator is completed, verify that the trusted CA environment variable is added to the `external-dns-operator` deployment by running the following command:
```

Comment:
Not critical, but this could be added as a step (unnumbered) in a .Verification section.

Reply:
Addressed